flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -96,7 +96,7 @@ under the License.
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
-				<version>2.3</version>
+				<version>2.4.1</version>
```

Comment:
updating the maven-shade-version in the quickstarts is a good idea. They are not inheriting the version from the flink-parent pom.
so the change is good !